ui/src/app/utils/access-utils.tsx

```diff
@@ -662,3 +662,15 @@ export const getStatusText = (status: AccessModuleStatus) => {
     ? `Completed on: ${displayDateWithoutHours(completionEpochMillis)}`
     : `Bypassed on: ${displayDateWithoutHours(bypassEpochMillis)}`;
 };
+
+export const hasRtExpired = (profile: Profile): boolean => {
```

Comment:
looks good!  I like the chaining.

It occurred to me that we need another filter line for "can expire" - but we don't, actually.  `hasExpired()` only returns true if an expiration exists.